powertools-metrics/src/test/java/software/amazon/lambda/powertools/metrics/internal/LambdaMetricsAspectTest.java

```diff
@@ -71,13 +72,33 @@ void tearDown() {
 
     @Test
     public void metricsWithoutColdStart() {
-        try (MockedStatic<SystemWrapper> mocked = mockStatic(SystemWrapper.class)) {
+        try (MockedStatic<SystemWrapper> mocked = mockStatic(SystemWrapper.class);
+             MockedStatic<software.amazon.lambda.powertools.core.internal.SystemWrapper> internalWrapper = mockStatic(software.amazon.lambda.powertools.core.internal.SystemWrapper.class)) {
```

Comment:
Are these the same class? does it need to be fully qualified?

Reply:
No, not same classes. One is from EMF lib and qualified one from powertools.